src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -8,8 +8,8 @@
   },
   "OrchardCore": {
     "OrchardCore.Setup": {
-      "DefaultCulture": "",
-      "SupportedCultures": [ "en" ]
+      "DefaultCulture": "", // When using "" the system OS culture will be used
+      "SupportedCultures": [ "ar", "cs", "da", "de", "el", "en", "es", "fa", "fi", "fr", "he", "hr", "hu", "id", "it", "ja", "ko", "lt", "mk", "nl", "pl", "pt", "ru", "sk", "sl", "sv", "tr", "uk", "vi", "zh-CN", "zh-TW" ]
     }
```

Comment:
I think the intention here is to provide a list of all the supported cultures.
While this works if someone is building and running from the source code, it doesn't provide any defaults for anyone using NuGet packages, because this `appsettings.json` file is not copied as part of the templates.

With media we use provide default settings in the source code if nothing is configured in `appsettings.json`, and if it is configured in `appsettings.json` that will override the defaults.

Maybe I am wrong but it seems like it would make more sense to do the same here, otherwise this configuration we have set here is useless for most users.

Reply:
I think we want to support the culture that we have in Crowdin. So it makes sense to have it there for that matter. If someone wants a different culture than what we support in Crowdin then he can add it in it's own appsettings.json file set in his default tenant for example. And we could add this appsettings.json file in the web app template.

Reply:
Thanks @Skrypt updating the templates will work well for new projects.

I would still say that for most existing users they will not see these support cultures without knowing to update their `appsettings.json` in their existing project.

Would it make sense for this to be a `DefaultSupportedCultures` property in 
https://github.com/OrchardCMS/OrchardCore/blob/b3bb908c9abd414b528626f9135fe4efb3d70c9d/src/OrchardCore.Modules/OrchardCore.Setup/Startup.cs#L25

Similar to what is done with Media/AllowedFileExtensions
https://github.com/OrchardCMS/OrchardCore/blob/6cddd9f9904ff117ba82587f39fc76f49fac3cc3/src/OrchardCore.Modules/OrchardCore.Media/Services/MediaOptionsConfiguration.cs#L70

Then as more cultures are available via crowdin we just update the default and they show for everyone?

Maybe is not the intention though?